index.html

```diff
@@ -1382,6 +1375,62 @@ <h2>
             </li>
           </ol>
         </section>
+        <section>
+          <h3>
+            Processing the manifest without a document
+          </h3>
+          <p>
+            The [=processing a manifest=] steps are invoked by [[HTML]]'s
+            processing steps for the [^link^] element, but MAY also be invoked
+            by the user agent to process a manifest without an associated
+            document.
+          </p>
+          <p>
+            In this case, the user agent SHOULD ensure that, at least at some
+            point in the past:
+          </p>
+          <ul>
+            <li>there was at least one page which is [=same origin=] as
+            |document URL:URL| that has a [^link^] element
+            |linkElement:HTMLLinkElement| with a [^link/rel^] of
+            [^link/rel/manifest^] and a [^link/href^] that resolves to
+            |manifest URL:URL|, and that
```

Comment:
Nit: not sure if we have a concept of this... but it feels like we should say something like "a valid link type manifest"... or just a "link type "manifest"", as per HTML.

Reply:
HTML doesn't export a concept of "link type". Internally, they just link "link type" to the [#linkTypes](https://html.spec.whatwg.org/multipage/links.html#linkTypes) heading, but it isn't exported.

I think it's reasonable to keep this the way it is; it's just literally stating what it is instead of referring to a concept.